doozerlib/rpmcfg.py

```diff
@@ -447,6 +473,13 @@ def build_rpm(self, version, release, terminate_event, scratch=False, retries=3,
                 # Status defaults to failure until explicitly set by succcess. This handles raised exceptions.
             }
 
+            if len(self.targets) > 1:  # for a multi target build, we need to ensure all buildroots have valid versions of golang compilers
+                # Check if this is a golang package
+                out, _ = exectools.cmd_assert(["rpmspec", "-q", "--buildrequires", "--", self.specfile])
+                if any(map(lambda dep: dep.startswith("golang"), out.splitlines())):
```

Comment:
this is fine but i think it's clearer as
```suggestion
                if any(dep.startswith("golang") for dep in out.splitlines()):
```